doc/GettingStarted.md

```diff
@@ -122,6 +125,64 @@ In addition, Vitess requires the software and libraries listed below.
     ``` sh
     $ sudo apt-get install openjdk-7-jre
     ```
+    
+#### OS X
+
+1.  [Install Homebrew](http://brew.sh/).
+
+2.  On OS X, MySQL 5.6 has to be used, MariaDB doesn't work for some reason yet. It should be installed from Homebrew
+    (install steps are below).
+   
+3.  ZooKeeper is used as lock service.
+
+4.  Run the following commands:
+
+    ``` sh
+    brew install go automake libtool memcached python mercurial git bison curl wget homebrew/versions/mysql56
```

Comment:
Regarding the chown command - I was testing the whole process on a clean virtual machine, so the folder /usr/local didn't exist. Anyway, this is a Homebrew issue, it has nothing to do with Vitess, but I could mention it in the docs if you'd like.

Regarding pkg-config - I forgot to mention, that I had XCode installed, which probably already includes pkg-config (if not, I can check my command history, not sure now). I'll mention that in the docs.